Misc/Positions_Area Positions 2 External.txt

```diff
@@ -36,22 +39,17 @@ EISN_N_CTR:Shannon Control:122.970:SNN:SNN:EISN:CTR:EISN:CTR:6600:6667
 EISN_S_CTR:Shannon Control:135.600:SNS:SNS:EISN:CTR:EISN:CTR:6600:6667
 EISN_W_CTR:Shannon Control:125.870:SNW:SNW:EISN:CTR:EISN:CTR:6600:6667
 EISN_I_CTR:Shannon Information:134.270:SNI:I:EISN:CTR:EISN:CTR:6600:6677
-EIDW_N_CTR:Dublin Control:129.170:DN:D:EIDW:CTR:EIDW:CTR::
-EIDW_S_CTR:Dublin Control:124.650:DS:D:EIDW:CTR:EIDW:CTR::
-EIDW_S_CTR:Dublin Control:126.250:DS:D:EIDW:CTR:EIDW:CTR::
-EIDW_S_CTR:Dublin Control:120.050:DS:D:EIDW:CTR:EIDW:CTR::
-EIDW_H_CTR:Dublin Control:119.550:DH:D:EIDW:CTR:EIDW:CTR::
-EIDW_N_APP:Dublin Control:129.170:DN:D:EIDW:APP:EIDW:CTR::
-EIDW_S_APP:Dublin Control:124.650:DS:D:EIDW:APP:EIDW:CTR::
-EIDW_S_APP:Dublin Control:126.250:DS:D:EIDW:APP:EIDW:CTR::
-EIDW_S_APP:Dublin Control:120.050:DS:D:EIDW:APP:EIDW:CTR::
-EIDW_H_CTR:Dublin Control:119.550:DH:D:EIDW:APP:EIDW:CTR::
-EIDW_APP:Dublin Approach:121.100:DWA:D:EIDW:APP:EIDW:APP::
+EIDW_1_CTR:Dublin Control:129.170:DNH:D:EIDW:CTR:EIDW:CTR::
+EIDW_2_CTR:Dublin Control:132.570:DNL:D:EIDW:CTR:EIDW:CTR::
+EIDW_3_CTR:Dublin Control:124.650:DSH:D:EIDW:CTR:EIDW:CTR::
+EIDW_4_CTR:Dublin Control:126.250:DSL:D:EIDW:CTR:EIDW:CTR::
 
 ;Iceland
 BIRD_CTR:Reykavik Control:119.700:RK:RK:BIRD:CTR:-:-:4100:4177
 BIRD_E_CTR:Reykavik Control:126.750:RKE:RKE:BIRD:CTR:-:-:4100:4177
```

Comment:
The line for BIRD_E_CTR can be removed, since the line for BIRD_4_CTR will identify this correctly.